cluster/manifests/kube-dns/depl-kube-dns.yaml

```diff
@@ -0,0 +1,143 @@
+apiVersion: extensions/v1beta1
+kind: Deployment
+metadata:
+  name: kube-dns
+  namespace: kube-system
+  labels:
+    application: kube-dns
+    version: v1.9
+    kubernetes.io/cluster-service: "true"
+spec:
+  # replicas: not specified here:
+  # 1. In order to make Addon Manager do not reconcile this replicas parameter.
+  # 2. Default is 1.
+  # 3. Will be tuned in real time if DNS horizontal auto-scaling is turned on.
+  strategy:
+    rollingUpdate:
+      maxSurge: 10%
+      maxUnavailable: 0
```

Comment:
> why, because of drain?

Yes, I mean if we only have one running at the time, then we will always be unavailable for a little while, when updating the nodes.